wolfcrypt/src/evp.c

```diff
@@ -2354,6 +2354,18 @@ static enum wc_HashType wolfSSL_EVP_md2macType(const WOLFSSL_EVP_MD *md)
     return WC_HASH_TYPE_NONE;
 }
 
+static const WOLFSSL_EVP_MD* wolfSSL_macType2EVP_md(enum wc_HashType type)
+{
+    const struct s_ent *ent ;
+
+    for( ent = md_tbl; ent->name != NULL; ent++) {
```

Comment:
Space between 'for' and '('.
No space between '(' and 'ent'.

Reply:
Fix function above too.

Reply:
Fixed.